lib/configuration.ex

```diff
@@ -1,61 +1,80 @@
 defmodule Configuration do
+  alias Game.Board, as: Board
+  alias Player.Human, as: Human
+  alias Player.Computer, as: Computer
 
-  def configure_game do
-    {player_1, player_2} = set_players()
-    board_size = get_board_size()
+  def configure_game(configuration \\ Configuration, console \\ Console) do
+    {player_1, player_2} = configuration.get_players(console)
+    board_size = configuration.get_board_size(console)
     %Game{board: Board.create(board_size), 
           size: board_size,
           player_1: player_1,
           player_2: player_2,
           current_player: player_1}
   end
 
-  def set_players(console \\ Console) do
-    game_type_input = console.get_game_type
-    if game_type_input |> Validator.is_valid_option? do
-      game_type_input 
-      |> Integer.parse 
-      |> elem(0) 
-      |> get_marks()
-    else
-      %Message{}.invalid |> console.print
-      set_players()
+  def get_board_size(console \\ Console) do
+    board_size_input = console.get_board_size
+    board_size_input 
```

Comment:
You could potentially just replace these two lines with `console.get_board_size`, right?